### ISSUE TITLE
FlutterToast is running into error on upgrade to Flutter 3.10

### ISSUE THREAD

**Issue by applegrew** [ponnamkarthik/FlutterToast#437]
Using this library gives error "Toastify" does not exist.

When it loads for the first time I see the following JS error.

    caught TypeError: Cannot read properties of null (reading 'exports')
    at toastify.js:14:48
 
The reason is the code inside toastify.js

    "object" == typeof module && module.exports ? module.exports = o() : t.Toastify = o()

Not sure what was the scenario before but in Flutter 3.10 `module` is coming as `null` and `null` is `object` so the `"object" == typeof module` check passes and it proceeds to `module.exports`, which naturally fails and `Toastify` remains undefined.

Maybe before Flutter 3.10 `module` was `undefined`?

I am using `fluttertoast 8.2.1`

**Comment by rakib205** [comment]
same here

**Comment by iliyami** [comment]
Same issue

**Comment by DamianB-BitFlipper** [comment]
Ditto

**Comment by rakib205** [comment]
@applegrew

**Comment by Mmisiek** [comment]
same here

**Comment by Mmisiek** [comment]
Anyone managed to fix it ?

**Comment by shatanikmahanty** [comment]
I found this difference.

Before flutter 3.10:

 "object" == typeof module in js console chrome debugger returned false

But in 3.10 it returns true.

So in previous versions module.exports was never called

**Comment by applegrew** [comment]
> I found this difference.
> 
> Before flutter 3.10:
> 
> "object" == typeof module in js console chrome debugger returned false
> 
> But in 3.10 it returns true.
> 
> So in previous versions module.exports was never called

Yes because before `module` was `undefined` but now some code is setting it to `null`.

**Comment by Aziz-cs** [comment]
Same issue, Any solutions?
